SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -75,12 +112,38 @@ void WellKnownEntitySystem::ProcessComponentUpdate(const Worker_ComponentId Comp
 	}
 }
 
-void WellKnownEntitySystem::ProcessComponentAdd(const Worker_ComponentId ComponentId, Schema_ComponentData* Data)
+void WellKnownEntitySystem::SaveSnapshotPartitionAuthServerSystemEntity()
+{
+	const ComponentData* SnapshotPartitionData =
+		GdkSubView->GetView()[SpatialConstants::INITIAL_SNAPSHOT_PARTITION_ENTITY_ID].Components.FindByPredicate([](const auto& CompData) {
+			return CompData.GetComponentId() == SpatialConstants::PARTITION_COMPONENT_ID;
+		});
+
+	if (SnapshotPartitionData == nullptr)
+	{
+		UE_LOG(LogWellKnownEntitySystem, Error,
+			   TEXT("Failed to store snapshot partition auth server. If that server crashes, we won't be able to restart it."));
+		return;
+	}
+
+	SnapshotPartitionServerSystemEntity = Schema_GetEntityId(Schema_GetComponentDataFields(SnapshotPartitionData->GetUnderlying()), 1);
```

Comment:
SYSTEM_WORKER_ID_FIELD ?

Reply:
Different component, but I can create a const for it: `PARTITION_SYSTEM_ENTITY_ID_FIELD`